app/services/scrapers/TulsaCity/tulsa_inmates.rb

```diff
@@ -0,0 +1,43 @@
+require 'byebug'
```

Comment:
not blocking but I don't think you need byebye bug (if you do we should try to move it someplace more global later)